### PR TITLE
String manipulation 2

### PR DESCRIPTION
**Pull request by 16hsuisaac**
string-manipulation-2

### CODE REVIEW

File: string-manipulation-2/src/capitalize-words.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    if (string[i - 1] === ' ') {
+      partString = partString + string.charAt(i).toUpperCase();
+    } if (string[i] !== ' ' & string[i - 1] !== ' ') {
+      partString = partString + string[i];
+    } if (string[i] === ' ') {
+      partString = partString + ' ';
+      newString = newString + partString;
+      partString = '';
+    } if (i === (string.length - 1)) {
+      newString = newString + partString;
```
Comment (codingcodymiller) [inline string-manipulation-2/src/capitalize-words.js:17]: Are these meant to be `else if` statements? They're formatted as if they are meant to be, but they are all just independent `if` statements currently.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work on this!! One note for you:

File: string-manipulation-2/src/capitalize-words.js (hunk 1 above)
Comment (16hsuisaac) [reply string-manipulation-2/src/capitalize-words.js:17]: Hey Cody! I actually meant for them to be independent if statements.  Is that bad practice to do so?

**Review by codingcodymiller — APPROVED** [review]
This works!! As long as they are on their own separate lines, this is okay. By starting the if statement on the same line as the closing curly for the previous block, it is deceptively similar to an else if statement.

**Pull request merged**